### ISSUE TITLE
Feature request: Use xmr.to's onion node when Orbot is running

### ISSUE THREAD

**Issue by Engelberg** [m2049r/xmrwallet#303]
xmr.to runs an onion node: http://xmrto2bturnore26.onion/ so it would be nice if monujo's integrated use of xmr.to would use the onion node if Orbot is running.

**Comment by leafcutterant** [comment]
I know @m2049r asked you to open a separate issue for this, but is this really an xmr.to-specific issue? Once Monerujo an connect to tor nodes (#100), any tor node's address can be added to the wallet and used.

**Comment by Engelberg** [comment]
It is an xmr.to specific issue because there is no way to configure the address of xmr.to.

**Comment by leafcutterant** [comment]
Oh, so you mean *accessing* xmr.to's service through their tor hidden service when Orbot is running? That would indeed be great.

By reading "node", I thought you mean that xmr.to runs a *Monero* node at http://xmrto2bturnore26.onion/ and you want Monerujo to connect to that. People usually call these things "hidden services" or "tor addresses/sites". But never mind, my interpretation may be even more mistaken because xmr.to probably doesn't run a Monero node that can be used as a remote node.

**Comment by SamsungGalaxyPlayer** [comment]
@leafcutterant yes, "onion node" is perhaps misleading wording. XMR.TO indeed runs a Tor version of its site at the provided address, not a full onion node. It would be great if Monerujo performed the API call to XMR.TO over Tor at this address when Orbot is running.

**Comment by kykeon** [comment]
I agree, it only makes sense to use the .onion if the wallet is going through Orbot VPN (=Tor)
